e2e_test/source_inline/kafka/shared_source.slt

```diff
@@ -158,6 +158,7 @@ internal_table.mjs --name s0 --type source
 # Note: heredoc in loop in mac's sh is ok, but not in linux's sh. So we use bash here.
 system ok
 bash -c 'for i in {0..9}; do
+sleep 0.1
```

Comment:
My theory is that: maybe consecutive `produce` will be consumed by risingwave in one batch. Introduce a little sleep here can separate the batches, and thus backfill has more chances to be faster than upstream...